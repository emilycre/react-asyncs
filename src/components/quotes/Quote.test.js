import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote Component', () => {
  it('Renders a Quote', () => {
    const quote = {
      character: 'Fry',
      quote: '...',
      image: '/image'
    };
    const wrapper = shallow(<Quote quote={quote} />);
    expect(wrapper).toMatchSnapshot();
  });
});
