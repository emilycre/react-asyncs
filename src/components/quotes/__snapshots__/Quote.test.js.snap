// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component Renders a Quote 1`] = `
<section>
  <img
    src="/image"
  />
  <p>
    Fry
  </p>
  <p>
    ...
  </p>
</section>
`;
